### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by hainv-2854**
> ## Related Tickets
> * [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/44701)
> 
> ## WHAT (optional)
> * Init project
> * Config i18n
> * Setup CI Local

### CODE REVIEW

**Comment by hainv-2854** [comment]
image

**Comment by hainv-2854** [comment]
ready

File: test/channels/application_cable/connection_test.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require "test_helper"
```
Comment (vanvtt-0952) [inline test/channels/application_cable/connection_test.rb]: folder test ko push lên gihub cu

**Comment by hainv-2854** [comment]
ready

**Pull request merged**